src/inputs/select.scss

```diff
@@ -64,3 +66,53 @@
     }
   }
 }
+
+@mixin iui-multi-select-tag-container {
+  position: absolute;
+  inset: 0 $iui-icons-default + $iui-l 0 $iui-sm;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  gap: $iui-xs;
+}
+
+@mixin iui-multi-select-tag {
+  @include iui-tag;
+  margin: 0;
+  display: inline-flex;
+  align-items: center;
+  height: 80%;
+  max-height: $iui-baseline * 3;
+
+  @at-root .iui-small#{&} {
```

Comment:
I think it is needed, because this weird selector is generated otherwise
![image](https://user-images.githubusercontent.com/81580355/166223995-c81d51e9-08f7-4303-9a11-c7927a34973b.png)

Versus
![image](https://user-images.githubusercontent.com/81580355/166223921-873dda5f-03d2-4515-8b25-b6b37db5e0ad.png)

Reply:
This was changed